velox/type/Variant.cpp

```diff
@@ -517,6 +536,10 @@ uint64_t variant::hash() const {
       auto dateValue = value<TypeKind::DATE>();
       return folly::Hash{}(dateValue.days());
     }
+    case TypeKind::DECIMAL: {
+      auto decimalValue = value<TypeKind::DECIMAL>();
+      return folly::Hash{}(decimalValue.toString());
```

Comment:
This should use the internal integer representation of Decimal... Using String is too inefficient.